### PR TITLE
Add Dootie exceptions

### PR DESCRIPTION
**Pull request by tjhanley**
- return some dootie when things go bad
  :hankey: 

@BrainScraps @rheaton

### CODE REVIEW

File: lib/duty_calculator/calculation.rb [hunk 1]
```diff
@@ -37,12 +38,17 @@ def transform_params(params)
...
     def self.get(params={})
       transformed_params = transform_params(validate_params(params))
+      # initialize connection
       conn = DutyCalculator::Client.new
+
       resp = conn.get "#{DutyCalculator::Client.api_base}/calculation", transformed_params
       hashed_resp = DutyCalculator::Response.new(resp.to_hash)
       resp = Hashie::Mash.new(hashed_resp)
-      raise Exception, "Duty Calculator Error: #{DutyCalculator::ErrorMessages.for_code(resp.body.error.code)}" if resp.body.error
-      raise Exception, "HTTP Status Code #{resp.status}" if resp.status.to_i != 200
```
Comment (tjhanley) [inline lib/duty_calculator/calculation.rb:45]: I did all i could to try and get them to return anything other than something that was handled by the first error and it didn't work... so i removed this

**Comment by rheaton** [comment]
@tjhanley The build is failing because of the ¯\_(ツ)_/¯ hehe

**Comment by tjhanley** [comment]
:(( i wanted to use that

File: lib/duty_calculator/dootie.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# encoding: utf-8
```
Comment (rheaton) [inline lib/duty_calculator/dootie.rb:1]: :+1:

**Pull request merged**